src/System.Private.ServiceModel/src/System/IdentityModel/Tokens/SamlSerializer.cs

```diff
@@ -2,15 +2,249 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
-
-using System.IdentityModel;
-
 namespace System.IdentityModel.Tokens
 {
+    using ServiceModel;
+    using System;
+    using System.Collections.Generic;
+    using System.Collections.ObjectModel;
+    using System.IdentityModel;
+    using System.IdentityModel.Selectors;
+    using System.Runtime.Serialization;
+    using System.Security;
+    using System.Security.Cryptography;
+    using System.Security.Cryptography.X509Certificates;
+    using System.Text;
+    using System.Xml;
```

Comment:
Done

---
In reply to: [78143615](https://github.com/dotnet/wcf/pull/1515#discussion_r78143615) [](ancestors = 78143615)